### DISCUSSION TITLE
A way to inspect requests (like selenium wire)?

### DISCUSSION THREAD

**Discussion by Firelord710** [apify/crawlee-python#492] [Q&A] [answered]
I had read on Medium that Crawlee included a way to inspect requests made by the browser (like graphql api calls), but didn't know such functionality existed and wasn't able to find anything else on it. 

Can anyone point me in the right direction ?

**Comment by janbuchar** [comment, marked as answer]
Huh, care to share the article? Regarding inspecting browser requests, this can be done via Playwright - https://playwright.dev/python/docs/network#network-events
